src/widgets/WithWhoisProtectPosition.php

```diff
@@ -31,28 +32,43 @@ public function init()
     public function run()
     {
         $currentPositions = $this->cart->getPositions();
-        $calculationId = $this->relatedPosition->getCalculationModel()->calculation_id;
+        $calculationId = $this->relatedPosition->getId();
         $price = $this->cart->formatCurrency($this->relatedPosition->cost, $this->relatedPosition->currency);
 
         $checkboxId = mt_rand();
         $parentExists = ArrayHelper::getColumn($currentPositions, 'parent_id');
+        $toCart = Json::htmlEncode(Url::toRoute('/cart/cart/index'));
```

Comment:
Сбило с толку название переменной, когда читал.
```suggestion
        $cartUrl = Json::htmlEncode(Url::toRoute('/cart/cart/index'));
```